docs/src/overview.md

```diff
@@ -0,0 +1,112 @@
+```@meta
+DocTestSetup = :(using Ferrite)
+```
+
+# Documentation of examples
+
+On this page, you find a documentation for each example currently available for `Ferrite`. The collection contains programs displaying different aspects and techniques of the `Ferrite` toolbox. 
+The programs are sorted as
+
+* A complete list shortly summarizing the topic, what they teach and which keywords related to the task they contain
+* Grouped by topic 
+
+## List of examples (chronological order)
```

Comment:
```suggestion
## List of examples
```
I think the list order looks good, but e.g. postprocessing was added after the von Mises example.
Super nice work @blaszm - I need to study that topology example closer - the method does look very neat!
(Maybe it could make sense to move "Nearly Incompressible Hyperelasticity" after "Hyperelasticity" since it follows from "Incompressible elasticity" and "Hyperelasticity"?)

Reply:
Ah right, the  "Nearly Incompressible Hyperelasticity" example should also be advanced, right?